omero/sysadmins/search.txt

```diff
@@ -199,20 +198,21 @@ servers of various sizes:
   - * Re-indexing type
     * Re-indexing duration
     * Binary repository size
-    * Indexer JVM settings
-    * :property:`omero.search.batch`
+    * Indexer memory settings
 
-  - * Background
-    * 31h
-    * 300GB
-    * -Xmx512M
-    * 50
+  - * Background [1]_
+    * 8h
+    * 19TB
+    * -Xmx4800m
 
-  - * Background
-    * 10 days
+  - * Off-line [2]_
+    * 6h30
     * 16TB
-    * -Xmx2048M
-    * 100
+    * -Xmx2g
```

Comment:
Yes that was the meaning of my `[2]` note. I initially put `--mem=2g` in the table but then thought it didn't render well the above line mentioning the JVM Indexer settings. Any suggestion?